app/konnect/updates.md

```diff
@@ -11,6 +11,8 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## March 2023
 
+**Custom OIDC Scope Requests**
+:Enterprise orgs can now configure additional OIDC scopes in their authorization request with Konnect. This allows organizations to specify additional scopes they may require so that Konnect can obtain a certain claim from their IdP. For example, the `upn` scope is required to retrieve the `userPrincipalName`from Azure.
```

Comment:
```suggestion
: Enterprise orgs can now configure additional OIDC scopes in their authorization request with {{site.konnect_short_name}}. This allows organizations to specify additional scopes they may require so that {{site.konnect_short_name}} can obtain a certain claim from their IdP. For example, the `upn` scope is required to retrieve the `userPrincipalName` from Azure.
```